### ISSUE TITLE
New Channel For Mature Players.

### ISSUE THREAD

**Issue by CD33KS** [drewhoener/JWSuggestions#168]
**Minecraft Username :** CD33KS

**My Idea :** 

Mature Channel for ingame.

**Reason :** 

The reason why I think a mature channel should be added to the server is that some people may have been muted in the past for simple words as "damn", "hell" or even by the slightest mistakes. **I was once muted for the reason of having a period being between a letter and a word, someone had asked me where I was from and I replied with "N.Ireland" little did I know that, that would reward me with a 20minute mute. I then played a little game of Charades with 3 people for the 20minutes trying to tell them what I was muted for. There was a staff member on at this time that decided to let me sit the mute after asking 3 people that didn't know what I was muted for that assumed I said the word "damn" so he/she let me sit the mute based off what he/she was told. At this time I had no idea that there was a command "/unmuteme <reason>" and was not made aware of the command until after I sat the 20minutes.** I would say this has happened to some people not necessarily the same thing but has happened.

**How I think it should work :** 

I think it should work as a "hidden channel" for those who are joining the server.  The player then could only see the server if they type something like "/cc m on" or somewhere along the lines of that and of course vice versa if they no longer wanted to see the channel "/cc m off".

I see there are a lot of "suggestions" submitted, some I wouldn't add myself but I also see a few that are in the current server which is nice to see infact. I also don't think this suggestion should be overlooked or put down in any way as I believe some, if not a lot of people would support the idea of a mature channel being implemented into the server.

**Thank you for your time
-CD**

**Comment by drewhoener** [comment]
Well the entire point of the filter is that we're a family friendly server, and even if you're slightly older the point of having rules is that people conform to them
And for people to have to join the channel there would have to be some kind of age restriction, and people can easily lie about their age
I can consider moving some of our mutable words to the current system that 'damn' uses which is just removing it and not allowing it to be sent but that's a different topic

**Comment by Violaessence** [comment]
I agree the server is family friendly, And I do like that, i am happy you are willing to consider removing that word, however the topic of a channel for people to join in which they can use more adult language would allow for players who are older to go and talk freely without worrying about being muted for such, so i have an idea to fix all of it.

Make the Channel Invite only, players who are known to the server usually present their ages and with that staff can decide who is mature enough to handle that chat, simply put, players who are MATURE can be invited into the Mature section, I don't believe there should be an exact age someone has to be and it depends on how mature they are, and whether or not they would be offended by the language.

**Comment by CD33KS** [comment]
I have read and appreciate the consideration of removing some of the mutable words but as people get older there are different things available to those people with further rules ofcourse which I think should also be taken into consideration as older players don't want to be muted by a plugin that could mute them for a reason that is non-existent. If there are no staff online at that time then they're screwed. I think if the mature channel was to be added there should be obvious rules such as no insulting each other and to use common sense when using the mature channel. To confirm the age maybe talk with them on the discord server or teamspeak to confirm their age, then and only then give them access to use the command and speak within this channel with other people who were also confirmed. As right now you can't /msg others or even speak in /l without getting muted as I seen the word "zesty" being over used the other day and they had taken it to a party chat but was still muted for using it and it was considered "spam". I don't know who but I remember this conversation being taken placed in the donator chat in the past week.

Back to the age verification though, if the player refuses to come onto Discord or Teamspeak to confirm their age then they should be refused access until they do.  If they aren't 16-18+ but sound it, if they agree to some type of terms set for that channel it would be their own fault for what they see in my opinion, not that there will be anything bad put into the chat but just lets say new swears they learn or something offensive they pick up on it would be their own fault.

**Comment by johnnywoof** [comment]
While reading your comments @CD33KS , I noticed how you kept referring to the mute system. I think the mute system is your primary reason on why you want a mature chat channel.
Anyways, there is one major issue with this channel. Did you know that there is the trade, help, misc, and many more channels available? And more importantly, did you see anyone use the channels?
This is what I'm afraid is going to happen to the mature channel. No one is going to use it, especially since it's ignored by default.

**Comment by Violaessence** [comment]
Johnny i saw people been told off not even a couple of days ago for playing a guess the lyrics game in the games channel.

Misc? there is no point since Donator is what Misc is anyway

Trade is used, not as frequently as it should be but it is used

Help i agree however isn't really advertised

Mature would be like Donator is now, a random channel where people talk however are more able to be open with the language they use however it will not be exclusive to Donators.

Aslong as we say there is a mature channel it will be used, since it will be a way for players to talk without the worry of the mute filter for topics in which language may be used.

**Comment by CD33KS** [comment]
I understand what you mean when you say "the trade, help and misc channels" are ignored. That's because players prefer to just ask the question or trade something when they're looking to do so instead of typing in command after command. What I think will make people use the mature channel when they are accept or if they apply to be accepted. Is that the avoids the swear filter they can speak in a more mature manner without the fear of being muted by mistake, my aim is at the mute system as it's very strict and I feel it would be nice to have a channel that would avoid the channel. Of course advertising etc. still wouldn't be allowed I'm completely fine with that as I don't see myself hopping onto another server but it the fact that you can be muted for the smallest of things and I think a mature channel would fix that problem and would require minimum attention regards to staff watching the chat as only adults or mature enough players would be speaking in this channel.

**Comment by Selttiks** [comment]
Well, my opinion on this matter is that most people do not use the chat channels to their full potential. They don't got to misc for random stuff non server related, people don't always use party chats, and use survival or donator instead for more private conversations that lots of members don't care as much, the kpop conversations, or the memeing from some members that others dont appreciate as much. 

The idea of the servers is to stay family friendly. In my opinion a mature channel moves away from this, and makes it seem like its starting to move away from the family friendly. Yes there is lots of older people on the server, but it would be impossible to enforce age limits on a mature channel, on top of the fact that this is including not family friendly content, onto a family friendly server. We have the discord channel, which has some looser guidelines, that has some more mature conversations on voice chat. I don't see how it would benefit the server in keeping it family friendly, which has been the goal the entire time.

**Comment by CD33KS** [comment]
I had closed this suggestion thinking as a lot of you keep referring to the server as "Family Friendly" I understand this but then I was recently muted again for a simple typo, I think a Mature Channel could avoid this from ever happening again or even the swear filter being re-worked. This has happened far too often for a lot of people and I believe the staff have been ignoring the problem I had typed "Anyone interested doing a long.boring task for 1k?" in chat moments ago and was giving a "family friendly" mute because of this. I don't think the problem should go on ignored but the problem should actually be dealt with. I know all plugins are coded by two people - Red_stoner64 and Johnnywoof" I know this takes time and those two are currently working on bugs with other plugins but I strongly believe that this should not be ignored any longer. (Yes there is a staff member currently online - I had /unmuteme and I had to message someone off of Minecraft to ask the staff member to unmute me) If I never had this person off of Minecraft I would have to sit a full 15 minute mute or whatever mute the plugin decided to give me at that time without having a say in whether or not I should be unmuted or remain muted. If you refuse to add a Mature Channel to the game could you atleast possibly rework the language plugin as it's extremely strict and I'm a little tired of having to make sure what I type I wont be muted for it.

Thanks for reading - CD

**Comment by Violaessence** [comment]
I think we should remove the auto-mute filter, it's proven to mute wrongly asmuch as it mutes rightly, a system like that has to be re-considered in my opinion, i believe it should not insta-mute instead give a warning for all things and then mute if the word is repeated in a certain time-frame that seems alot fairer

**Comment by drewhoener** [comment]
62. It's 62 not 64

**Comment by CD33KS** [comment]
Fixed.

**Comment by Violaessence** [comment]
I'll refine my idea:

I believe that the filter should warn you like it does with some words, except applied to all things that now warrant an insta-mute.

If the word is said again in a timeframe which could be decided by you to what you believe is fair, then the person is muted, this avoids people accidentally saying a word or something that may seem like an IP and be insta muted for it, but of course stops any spammers or people who wish to say those words harmfully.

I believe this will make the system fairer and still control chat and stop people from been insta-muted by accidental reason.

**Comment by Violaessence** [comment]
Any Word on this?

**Comment by drewhoener** [comment]
You literally submitted the last comment 3 hours ago, I'm currently eating dinner, then shower, then go to bed xD

**Comment by Violaessence** [comment]
Okay sorry xD

**Comment by PianoPlayingHusky** [comment]
@CD33KS Sorry about that zesty thing, we were kinda hyper that time I guess ;-;
